lib/resources.js

```diff
@@ -249,6 +249,7 @@ module.exports = function (app, _private = null) {
             size: params.items_size,
             from: params.items_from,
             merge_checkin_card_items: params.merge_checkin_card_items,
+            removeElectronicResources: true,
```

Comment:
can this be something like removeElectronicResourcesFromItemsArray? maybe clunky but a little clearer

Reply:
👍